prime-router/settings/STLTs/MN/mn-doh.yml

```diff
@@ -0,0 +1,101 @@
+# To submit this to localhost, run:
+#
+# ./prime multiple-settings set --input ./settings/STLTs/MN/mn-doh.yml
+#
+- name: "mn-doh"
+  description: "Minnesota Department of Health"
+  jurisdiction: "STATE"
+  stateCode: "MN"
+  countyName: null
+  filters: null
+  featureFlags: null
+  keys: null
+  senders: []
+  receivers:
+    - name: "full-elr"
```

Comment:
There is a secondary active filter in their settings that needs to be moved over as well.

Reply:
The second active doesn't being used.  There is no transport for it.

Reply:
Got it, I missed that, thank you